javascript-arrays/main.js

```diff
@@ -0,0 +1,19 @@
+var colors = ['red', 'white', 'blue'];
+console.log('value of colors[0]:', colors[0]);
+console.log('value of colors[1]:', colors[1]);
+console.log('value of colors[2]:', colors[2]);
+console.log('America is ' + colors[0] + ', ' + colors[1] + ' and ' + colors[2] + '.');
+console.log('Mexico is ' + colors[0] + ', ' + colors[1] + ' and green.');
+console.log('value of colors:', colors);
+
+var students = ['Brian', 'Matthew', 'Juver', 'Salvador'];
+
+var numberOfStudents = students.length;
+console.log('There are ' + numberOfStudents + ' students in the class.');
+
+var lastIndex = numberOfStudents - 1;
+console.log('value of number of students -1:', lastIndex);
+var lastStudent = students[3];
```

Comment:
you should be using the `lastIndex` variable here to access the last item in the array.